server/resourcemanager/resource_manager_test.go

```diff
@@ -164,6 +200,89 @@ func TestSampleResource(t *testing.T) {
 	})
 }
 
+func TestRestrictedResource(t *testing.T) {
```

Comment:
This test validates the behaviour for "partial" resources (meaning, ones that don't implement the entire CRUD interface)

Reply:
Thinking about it that make sense to change the test name to something like `TestResourceWithPartialOperations`?
By looking at its name the first thing that I associated is that a `RestrictedResource` is a `private` one.